$NetBSD: patch-aa,v 1.3 2011/10/03 23:45:27 schnoebe Exp $

Move the configuration file up a directory, no point in having
a directory containing one file.

--- configure.ac.orig	2011-06-12 10:49:35.000000000 +0000
+++ configure.ac
@@ -275,7 +275,7 @@ fi
 
 # where to find the configuration file
 # TODO: this should probably be fixed to be nicer
-CONFIGFILE=`eval echo $sysconfdir/cvsd/cvsd.conf | \
+CONFIGFILE=`eval echo $sysconfdir/cvsd.conf | \
             sed "s%^NONE/%${prefix}/%" | \
             sed "s%^NONE/%${ac_default_prefix}/%"`
 AC_SUBST(CONFIGFILE)
